links.opam

```diff
@@ -60,7 +60,7 @@ depends: [
   "base64"
   "linenoise"
   "ANSITerminal"
-  "lwt"
+  "lwt" {>= "3.1.0"}
```

Comment:
Why do you need to constrain LWT?? Do we have some other dependency that depends on 3.1.0?